main.py

```diff
@@ -1,10 +1,41 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as players:
+        players_data = json.load(players)
+
+    for player_nickname, player_attributes in players_data.items():
+        race, created = Race.objects.get_or_create(
+            name=player_attributes["race"]["name"],
+            description=player_attributes["race"]["description"]
+        )
+        if created:
+            for skill in player_attributes["race"].get("skills"):
```

Comment:
```suggestion
            for skill in player_attributes["race"].get("skills", []):
```
Possible error if not use default attribute